src/utils/listJsonInDirectory.js

```diff
@@ -0,0 +1,6 @@
+import fs from 'fs';
+
+export default (path) => {
```

Comment:
Unify your export style, In this case you used arrow function, in this https://github.com/vviital/json-simple-formatter/pull/1/files?diff=unified#diff-c6f962d13812bb52f0b3b35783200728R10 anonymous function